wrappers/ortools.rb

```diff
@@ -136,20 +135,15 @@ def solve(vrp, job, thread_proc = nil, &block)
       routes = []
       services_positions = { always_first: [], always_last: [], never_first: [], never_last: [] }
       vrp.services.each_with_index{ |service, service_index|
-        vehicles_indices =
-          if service.skills.any? && vrp.vehicles.all?{ |vehicle| vehicle.skills.empty? } &&
-             service.unavailable_days.empty?
-            []
-          else
-            vrp.vehicles.collect.with_index{ |vehicle, index|
-              if (service.skills.empty? || !vehicle.skills.empty? &&
-                 ((vehicle.skills[0] & service.skills).size == service.skills.size) &&
-                 check_services_compatible_days(vrp, vehicle, service)) &&
-                 (service.unavailable_days.empty? || !service.unavailable_days.include?(vehicle.global_day_index))
-                index
-              end
-            }.compact
-          end
+        vehicles_indices = []
+        detect_unfeasible_services(vrp) if service.vehicle_compatibility.nil?
+        vrp.vehicles.each_with_index{ |vehicle, index|
+          next unless (service.vehicle_compatibility[vehicle.id].nil? || service.vehicle_compatibility[vehicle.id]) &&
```

Comment:
Could we agree on a rubocop rule to avoid `unless` for complex conditions ?

Reply:
:100: All for it, don't know if there is a way, do you know if not will check

Reply:
I think this rule meet our expectation: https://docs.rubocop.org/rubocop/cops_style.html#styleunlesslogicaloperators